.gitignore

```diff
@@ -23,3 +23,5 @@ valgrind-python.supp
 *.pyd
 lcov.info
 netlify_build/
+# pycharm files
+/.idea/
```

Comment:
Personally I think that this is irrelevant for this `.gitignore`, e.g. we could add ignores for `.vscode` and every editor under the sun, but we don't. I think it seems more appropriate to add this to your global `.gitignore`, would you be amenable to removing it here (and adding it to your global `.gitignore` if suits)?

Reply:
:+1:

Reply:
done.